pkg/networkmanager/networking.css

```diff
@@ -268,21 +272,69 @@ h1 .onoff-ct {
     }
 }
 
-#zones-listing,
-#services-listing {
-    margin: 2rem 0;
+#firewall-heading,
+#networking-graphs {
+    background: var(--color-white);
+    padding: 1rem 3rem;
+}
+
+#networking-graphs .row {
+    margin: 0;
 }
 
-#services-listing .listing-ct-body h5 {
-    margin: 1.5rem 0 1rem;
+#firewall-heading {
+    margin-bottom: 2rem;
 }
 
-#services-listing .listing-ct-body ul {
-    padding-left: 1rem;
+#firewall-heading > ol.breadcrumb {
+    margin-bottom: 0;
 }
 
-#services-listing .listing-ct-body li + li {
-    margin-top: 1rem;
+#firewall-heading > h1 {
+    display: flex;
+    align-items: stretch;
+    justify-content: space-between;
+    margin-top: 0;
+}
+#firewall-heading > h1 > span:first-child {
+    display: flex;
+}
+
+#firewall-heading .btn-group {
+    display: inline-flex;
+    align-items: center;
+}
+
+#zones-listing {
+    margin-top: 0;
+    padding-top: 0;
+}
+
+.zone-section {
+    background: var(--color-white);
+    border-top: 1px solid var(--color-border-default);
+    border-left: 1px solid var(--color-border-default);
+    border-right: 1px solid var(--color-border-default);
+    margin: 0 0 2rem 0;
+    box-shadow: var(--pf-global--BoxShadow--sm);
+}
+
+.zone-section > h4:first-child {
+    margin: 0;
+    padding: 0.5rem 0.75rem;
+    border-bottom: 1px solid var(--color-border-default);
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.zone-section > h4 > span:last-child {
+    display: inline-flex;
+    align-content: center;
+}
+
+.zone-section > h4:first-child > span > span {
```

Comment:
This... is odd.

1. The interface labels shouldn't be under H4 (this is even an accessibility issue)
2. Unclassed spans (and divs) shouldn't be in CSS selectors